beetsplug/acousticbrainz.py

```diff
@@ -164,6 +165,7 @@ def _get_data(self, mbid):
     def _fetch_info(self, items, write, force):
         """Fetch additional information from AcousticBrainz for the `item`s.
         """
+        tags = self.config['tags'].get()
```

Comment:
One last suggestion here: instead of `.get()`, use `as_str_seq()`. That has a couple of benefits:

- It produces an error early if the user enters the wrong kind of data here (for example, a number) instead of causing an unpredictable Python error later.
- It lets the user enter tags as a whitespace-separated string as an optional alternative to using YAML list syntax, which can be a bit friendlier.